cfripper/model/utils.py

```diff
@@ -101,3 +104,15 @@ def get_managed_policy(managed_policy_arn):
             }
         )
     return None
+
+
+def get_account_id_from_arn(arn) -> Optional[str]:
+    match = REGEX_ARN.match(arn)
+    if match:
+        return match.group(3)
+
+
+def get_account_id_from_iam_arn(arn) -> Optional[str]:
+    match = REGEX_IAM_ARN.match(arn)
+    if match:
+        return match.group(1)
```

Comment:
In principals, specifying only the account id in an arn is equivalent to `arn:aws:iam::<account_id>:root`, this case should also be considered

https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_principal.html